addons/stock/tests/test_quant.py

```diff
@@ -818,6 +819,133 @@ def test_quant_display_name(self):
                 self.assertEqual(q.display_name, '%s - %s' % (q.location_id.display_name, q.lot_id.name))
             else:
                 self.assertEqual(q.display_name, '%s' % (q.location_id.display_name))
+    def test_relocate(self):
+        """ Test the relocation wizard. """
+        def _get_relocate_wizard(quant_ids):
+            relocate_wizard_dict = quant_ids.action_stock_quant_relocate()
+            return Form(self.env[relocate_wizard_dict['res_model']].with_context(relocate_wizard_dict['context']))
+
+        self.env.user.write({'groups_id': [(4, self.env.ref('stock.group_tracking_lot').id)]})
+        package_01 = self.env['stock.quant.package'].create({})
+        package_02 = self.env['stock.quant.package'].create({})
+        self.env['stock.quant']._update_available_quantity(self.product, self.stock_location, 10, package_id=package_01)
+        quant_a = self.env['stock.quant'].search([('product_id', '=', self.product.id)])
+
+        # testing assigning a package to a quant
+        relocate_wizard = _get_relocate_wizard(quant_a)
+        relocate_wizard.dest_package_id = package_02
+        relocate_wizard.save().action_relocate_quants()
+        new_quant_a = self.env['stock.quant'].search([('product_id', '=', self.product.id), ('quantity', '=', 10)])
+        self.assertEqual(new_quant_a.package_id, package_02)
+
+        # testing moving a packed quant to a new location
+        relocate_wizard = _get_relocate_wizard(new_quant_a)
```

Comment:
We have to check when `is_not_full_package` should be false too to be sure the field is correctly computed (and not always true)
```suggestion
        relocate_wizard = _get_relocate_wizard(new_quant_a)
        self.assertEqual(relocate_wizard.is_not_full_package, False)
```